tracer/test/snapshots/AspNetMvc5Tests.InferredProxySpans_Enabled.__path=_BadRequestWithStatusCode_401-TransferRequest=true_statusCode=401.verified.txt

```diff
@@ -0,0 +1,107 @@
+[
```

Comment:
The snapshots with inferred proxy spans enabled and NOT showing the api gateway span.

Reply:
Nope, I left this comment to myself to fix the snapshots. The `aws.apigateway` spans are there now, so I'll resolve this thread. Thanks!